scientist-softserv/harvard-curiosity
# Order exhibits table, pagination realignment, and custom 404 error page

**Order exhibits table, pagination realignment, and custom 404 error page**
* * *

**JIRA Ticket**:
* #248

# What does this Pull Request do?
Resolves a few tasks on the "lower priority" ticket, related to:
* Order exhibits table
* Pagination realignment
* Custom 404 page

# How should this be tested?
* Navigate to the ["order exhibits" tab](https://localhost:21407/exhibits/edit) of the admin panel and resize the screen; the titles now wrap and table should never overlap with the Actions menu
* Navigate to a search results page with multiple pages; the pagination now aligns with the search results column (rather than spanning the entire width of the page)
* A [custom 404 error page](https://localhost:21407/error) has been created to match the styling of Library.Harvard (https://library.harvard.edu/collections-exhibits/explore-collectionsaskf), except the search box has been removed and the prompt suggests that users return back to Curiosity homepage. (Note: in order to see this locally, you'll have to add `RAILS_ENV=development` to your .env and then switch to `config.consider_all_requests_local = false` in [config/environments/development.rb](https://github.com/scientist-softserv/harvard-curiosity/blob/main/config/environments/development.rb#L21))
   * HL logo links to Library.Harvard
   * Curiosity link points to the curiosity homepage

Page not found:
![Screenshot 2022-12-05 at 16-50-34 The page you were looking for doesn't exist (404)](https://user-images.githubusercontent.com/13121501/205749882-092d2754-5e5a-49b4-86a1-376489963d96.png)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? NA
- integration tests? NA

# Interested parties
@phil-plencner-hl @dl-maura
